components/PrimaryButton/PrimaryButton.tsx

```diff
@@ -0,0 +1,7 @@
+import { PrimaryButton } from "@/components/PrimaryButton/PrimaryButton.styled.js";
```

Comment:
Sollen die styled components keine tsx-Datei sein? Ich würde sie einfach mit in diese Datei holen und dann die variable StyledPrimaryButton nennen. Das ist auch ein Nachteil, dass man immer zusätzliche Komponenten hat, für die man Namen braucht.